Library/Client/DisturbStarter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Vpg\Disturb\Client;
+
+use \Phalcon\Mvc\User\Component;
+use \Vpg\Disturb\Message;
+
+class DisturbStarter extends Component
```

Comment:
maybe rename `DisturbStarter` in `command` to make the use more readable : 
`\Disturb\Client\Command::start(...)`